build.gradle

```diff
@@ -118,14 +118,9 @@ uploadArchives {
             }
             developers {
                 developer {
-                    id 'jbwyme'
-                    name 'Josh Wymer'
-                    email 'josh@mixpanel.com'
-                }
-                developer {
-                    id 'joeatwork'
-                    name 'Joe Bowers'
-                    email 'joe@mixpanel.com'
+                    id 'mixpanel_dev'
+                    name 'Mixpanel Developers'
+                    email 'dev@mixpanel.com'
```

Comment:
Actually, I can just change this to `dev+android@mixpanel.com` and folks can filter it appropriately